<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VistaFumetto</class>
 <widget class="QWidget" name="VistaFumetto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QLabel" name="l_categoria">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_distributore">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_editore">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_collana">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_sottocollana">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_prezzo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_quantita">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_modifica">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifica</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_elimina">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Elimina</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_titolo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
